### DISCUSSION TITLE
Total infectious number more that population

### DISCUSSION THREAD

**Discussion by Goonigh2020** [neherlab/covid19_scenarios#473] [General]
Good day again,
I ran the simulation and downloaded covid.results.deterministic.csv to see the result.
When I summarize all infectious number for the whole period (april december) it gives me total infectious figure twice more than my total population.

Can you please advice whether I am doing something wrong?

**Comment by noleti** [comment]
People stay infectious for about 3 days, so if you sum the 'current infectious' for each day, you will count individual people three times.

**Reply by Goonigh2020** [reply]
Thank you for the prompt response. If I get the difference between figures for "susceptible" at the start and end dates of the period, will I get the predicted number of total cases? Is there any way to calculate that figure (total cases for the period)?

**Reply by nnoll** [reply]
The sum of the cumulative recovered and cumulative fatalities at the end of the simulation will give you the cumulative number of infections. Alternatively, take the initial susceptible size and subtract off the current susceptible

**Reply by Goonigh2020** [reply]
Thank you!
